.github/workflows/verify-ghas.yaml

```diff
@@ -195,21 +191,19 @@ jobs:
         with:
           name: cookiecutter_outputs
 
-#      - name: "Change directory"  # Have to CD here to make sure this works
-#        shell: bash
-#        run: |
+      #      - name: "Change directory"  # Have to CD here to make sure this works
+      #        shell: bash
+      #        run: |
       #          cd prj_${{ matrix.license }}_2_${{ matrix.rtd }}
 
-      # More info on options: https://github.com/conda-incubator/setup-miniconda
-      - uses: conda-incubator/setup-miniconda@v2
+      # More info on options: https://github.com/marketplace/actions/provision-with-micromamba
+      - uses: mamba-org/provision-with-micromamba@main
         with:
-          python-version: ${{ matrix.python-version }}
           environment-file: prj_${{ matrix.license }}_2_${{ matrix.rtd }}/devtools/conda-envs/test_env.yaml
-
-          activate-environment: test
-          auto-update-conda: false
-          auto-activate-base: false
-          show-channel-urls: true
+          environment-name: test
+          channels: defaults
```

Comment:
Out of curiosity, are we aware of anything that's still pulled down from `defaults` (and not available on `conda-forge`)? I know years ago there was not parity but I think in recent years that's been achieved, if not inverted.

Happy to open a separate issue if this is not a simple topic

Reply:
I think we shouldn't have it listed here, there shouldn't be anything missing. If users do need it, they can put channels in their `environment.yaml` that they need.

Reply:
I have not heard one way or the other. This was in place due to precedence and to mimic the behavior when people have their conda environment set up. Given most (anecdotally) of the time people do `conda config --add channels conda-forge`, their conda envs will be `conda-forge,defaults` normally, so this replicated it. 

I'm fine removing the defaults channel though from conda-forge, especially if there is parity, but I would suggest doing that in a separate PR as there are several references to the "defaults as fallback" elsewhere in the code.